WooCommerce/Classes/ViewRelated/Orders/Refund/RefundSubmissionUseCase.swift

```diff
@@ -390,6 +398,21 @@ private extension RefundSubmissionUseCase {
         stores.dispatch(action)
         trackCreateRefundRequest()
     }
+
+    /// Retrieves the up-to-date refund data
+    /// - Parameters:
+    ///   - refund: the refund to retrieve details from.
+    ///   - onCompletion: called when the submission completes.
+    func getLatestRefundData(refund: Refund, onCompletion: @escaping (Result<Void, Error>) -> Void) {
+        let action = RefundAction.retrieveRefund(siteID: details.order.siteID, orderID: details.order.orderID, refundID: refund.refundID) {
+                (refundDataFromGETreq, error) in
```

Comment:
`refundDataFromGETreq`

There are 2 hard things in programming: naming, cache invalidation, and off by one errors ;)

I think this is a bit wordy and, and the view layer shouldn't care where the refund data is coming from.

You're also not (really) using it for anything, so just make this
```suggestion
                (_, error) in
```

Reply:
Oops! This one flew under the radar in the process of testing -> cleaning things up for the PR 😅 Sorry for that!